src/vivarium_gates_mncnh/components/observers.py

```diff
@@ -96,6 +98,28 @@ def register_stratifications(self, builder: Builder) -> None:
             is_vectorized=True,
             requires_columns=[COLUMNS.ULTRASOUND_TYPE],
         )
+        builder.results.register_stratification(
```

Comment:
These intervention stratifications just moved from being localized in an observer